.github/pull_request_template.md

```diff
@@ -13,6 +14,6 @@ Please include some guiding steps on how to test this change during a review.
 
 ## Checklist
 - [ ] Unit tests have been updated/added
-- [ ] PR is labeled accordingly
+- [ ] PR is labeled accordingly with a single label
```

Comment:
do we want a checklist item that the commit message documentation was read? 😄

Reply:
let's see we still can add more to it afterwards but for now we can keep it like that i think :)